_projects/meta_review.md

```diff
@@ -0,0 +1,95 @@
+---
+collaborating_projects:
+  - coala
+desc: "Implement a meta-review system that tracks and provides meaningful analysis of reactions on reviews."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - "https://github.com/coala/projects/issues/480"
+markdown: meta_review.md
+mentors:
+  - hemangsk
+  - blazeu
+  - alisianoi
+  - andrewda
+name: "Meta-review"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "The applicant should be familiar with GitHub GraphQL API and JavaScript."
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The meta-review project is about building a meta-review system,
+similar to a meta-moderation system, that can track emoji responses to
+review comments. These responses are to be collected, processed and displayed
+on [gh-board](https://github.com/coala/gh-board), which is a nice serverless
+kanban board.
+
+People including the author of pull request respond to comments by attaching
+emojis. Those emojis are called meta-review, or
+[reactions](https://developer.github.com/v4/enum/reactioncontent/). THUMBS_UP
+and THUMBS_DOWN, together with other reactions, are used by people especially
+the author of pull request to publicly acknowledge or reject review comments.
+This provides feedback to reviewers and all other members.
+
+However, those reactions spread over all PRs and are not collected and
+analyzed. By tracking reactions, a nice feedback loop would be created
+for both senior and junior reviewers. Also, this information can be used
+as newcomer metrics.
```

Comment:
We can add a line explaining what exactly do we imply by 'newcomer metrics'. Basically statistics related to newcomers and then figuring out strategies to improve their induction process and to  produce good code

Reply:
There is a project which focuses on newcomer gamification. Definitely, the newcomer metrics used in that project would not include meta-review statistics. IMO the statistics generated in this project could be used in that project in the future. Something like `pre-fetch.js` could be reused, but strategies to improve their induction process is irrelevant to this project.